stylesheets/base.css

```diff
@@ -55,3 +55,7 @@
 .relative {
     position: relative;
 }
+
+.color-red {
+    color: red;
+}
```

Comment:
you can use much more sort name, for an example say, clr-r or simply c-red!!
just a suggestion the rest of the code is good!

Reply:
okay done